sim/hunter/item_sets_pve_phase_8.go

```diff
@@ -0,0 +1,218 @@
+package hunter
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+	"github.com/wowsims/sod/sim/core/stats"
+)
+
+// TODO: UNCOMMENT WHEN TIER SETS ARE ADDED TO THE SIM
+// var ItemSetDawnstalkerProwess = core.NewItemSet(core.ItemSet{
+// 	Name: "Dawnstalker Prowess",
+// 	Bonuses: map[int32]core.ApplyEffect{
+// 		2: func(agent core.Agent) {
+// 			hunter := agent.(HunterAgent).GetHunter()
+// 			hunter.applyScarletEnclaveMelee2PBonus()
+// 		},
+// 		4: func(agent core.Agent) {
+// 			hunter := agent.(HunterAgent).GetHunter()
+// 			hunter.applyScarletEnclaveMelee4PBonus()
+// 		},
+// 		6: func(agent core.Agent) {
+// 			hunter := agent.(HunterAgent).GetHunter()
+// 			hunter.applyScarletEnclaveMelee6PBonus()
+// 		},
+// 	},
+// })
+
+// Your Strikes and Mongoose Bite deal 20% increased damage to targets afflicted with your Serpent Sting or Wyvern Strike.
+func (hunter *Hunter) applyScarletEnclaveMelee2PBonus() {
+	label := "S03 - Item - Scarlet Enclave - Hunter - Melee 2P Bonus"
+	if hunter.HasAura(label) {
+		return
+	}
+
+	damageMod := hunter.AddDynamicMod(core.SpellModConfig{
+		Kind:      core.SpellMod_DamageDone_Flat,
+		ClassMask: ClassSpellMask_HunterStrikes | ClassSpellMask_HunterMongooseBite,
+	})
```

Comment:
I actually realized that both this and the T2 Ranged 2p need to be updated. Conditional damage effects use a dummy script that we've found always seems to be equivalent to a % damage multiplier rather than additive. Both should actually use `core.SpellMod_DamageDone_Pct` and then `FloatValue: 1.0`, then in the handler updating the FloatValue conditionally to either 1.20 or 1.0